ocaml/xapi/vgpuops.ml

```diff
@@ -129,9 +129,8 @@ let reserve_free_virtual_function ~__context vm pf =
 let add_vgpus_to_vm ~__context host vm vgpus =
   (* Only support a maximum of one virtual GPU per VM for now. *)
   (try Db.VM.remove_from_other_config ~__context ~self:vm ~key:Xapi_globs.vgpu_pci with _ -> ());
-  match vgpus with
-  | [] -> ()
-  | vgpu :: _ ->
+  List.iter (fun vgpu -> debug "Vgpus located on VM(%s) are: %s" (Ref.string_of vm) (Ref.string_of vgpu.vgpu_ref)) vgpus;
+  List.iter (fun vgpu ->
```

Comment:
Two `iter`s?

Reply:
What is this change doing here anyway? This was already in another PR?

Reply:
yes,   you are right, 
This is involved as Min and I developed parallelly, 
I will delete it here and merge in my PR.